### ISSUE TITLE
Emacs "/" Keybinding not working

### ISSUE THREAD

**Issue by nerd-qp** [XmacsLabs/mogan#1678]
### Mogan Version

v1.2.5

### Operating System Version and Architecture

MacOS

### Describe Bug

Toggle on look and feel: Emacs. The Emacs "/" for `undo` does not register anything.

This behaviour does not happen for v1.1.x.

### Expected Behavior

Emacs "/" should `undo` the edits.

**Comment by darcy-shen** [comment]
The root cause is Qt 6 can not handle the `C-/` event. That's why "emacs /" does not work when "emacs" means Ctrl.

**Comment by nerd-qp** [comment]
Yep, sorry for the "emacs /" confusion. Thanks for tracking down the root cause!

Do you think one can bypass the restriction set by Qt 6?

**Comment by ProfFan** [comment]
Seems strange, do you have a reference where it said so in the upstream docs? @da-liii

**Comment by da-liii** [comment]
I tested via code but didn't find a reference from Qt official doc.
